.github/workflows/build.yml

```diff
@@ -825,20 +773,10 @@ jobs:
         run: |
           docker run --name=front-test --net=host -v $PWD/output:/output \
             ${{ fromJSON(needs.build.outputs.stable_tags).front-tests }} \
-            sh -c "npm run test-coverage && cp -R coverage /output"
+            sh -c "CI=true npm run test"
```

Comment:
Seems like `CI=true` is new, what does it do?

Reply:
I'm not fully certain, but it at least prevents the test runner from watching for file changes after running tests...

Reply:
Yeah, it's an equivalent to `vitest run`. I'd prefer to be more explicit and either use `npx vitest run`, or `npm run test run` (run run!).

Reply:
Gentle ping, has this comment been forgotten, or do you disagree?